terraform/modules/paas/variables.tf

```diff
@@ -23,11 +23,13 @@ locals {
   redis_service_name        = "register-redis-${var.app_environment}"
   web_app_name              = "register-${var.app_environment}"
   web_app_start_command     = "bundle exec rails db:migrate && bundle exec rails server -b 0.0.0.0"
-  app_environment_variables = {
+  app_environment = {
     AUTHORISED_HOSTS               = "${local.web_app_name}.london.cloudapps.digital"
     RAILS_ENV                      = "production"
     RACK_ENV                       = "production"
     RAILS_SERVE_STATIC_FILES       = true
     SECRET_KEY_BASE                = "TestKey"
   }
+  worker_app_start_command = "bundle exec sidekiq -C config/sidekiq.yml"
+  worker_app_name          = "register-bg-jobs"
```

Comment:
Shouldn't it include the name of the environment?

Reply:
do you mean something like---qa-register-bg-jobs?

Reply:
done...